static/chart.html

```diff
@@ -0,0 +1,237 @@
+        else {
+          arr.push(['sal',0]);}
+
+        if(params.chck=='1'){
+          arr.push(['chck',1]);}
+        else {
+          arr.push(['chck',0]);}
+
+        if(params.ched=='1') {
+          arr.push(['ched',1]);}
+        else {
+          arr.push(['ched',0]);}
+
+        if(params.fet=='1') {
+          arr.push(['fet',1]);} 
+        else {
+          arr.push(['fet',0]);}
+
+        if(params.asi=='1') {
+          arr.push(['asi',1]);}
+        else {
+          arr.push(['asi',0]);}
+
+        if(params.pro=='1'){
+          arr.push(['pro',1]);}
+        else {
+          arr.push(['pro',0]);}
+
+        if(params.Peppe=='1') {
+          arr.push(['Peppe',1]);}
+        else {
+          arr.push(['Peppe',0]);}
+
+        if(params.Oli=='1') {
+          arr.push(['Oli',1]);}
+        else {
+          arr.push(['Oli',0]);}
+
+        if(params.Garlic=='1') {
+            arr.push(['Garlic',1]);}
+        else {
+            arr.push(['Garlic',0]);}
+
+        if(params.Jala=='1') {
+            arr.push(['Jala',1]);}
+        else {
+            arr.push(['Jala',0]);}
+
+        if(params.Mush=='1') {
+            arr.push(['Mush',1]);} 
+        else {
+            arr.push(['Mush',0]);}
+        
+        if(params.Pine=='1') {
+            arr.push(['Pine',1]);}
+        else {
+            arr.push(['Pine',0]);}
+
+        if(params.Oni=='1') {  
+            arr.push(['Oni',1]);} 
+        else {
+            arr.push(['Oni',0]);}
+
+        if(params.red=='1') {
+            arr.push(['red',1]);}
+        else {
+            arr.push(['red',0]);}
+        
+        if(params.Spi=='1'){
+            arr.push(['Spi',1]);}
+        else {
+            arr.push(['Spi',0]);}
+
+        if(params.Tomat=='1'){
+            arr.push(['Tomat',1]);}
+        else {
+            arr.push(['Tomat',0]);}
+        
+        if(params.Dic=='1'){
+             arr.push(['Dic',1]);}
+        else {
+            arr.push(['Dic',0]);}
+
+        if(params.hot=='1'){
+                arrpush(['hot',1]);} 
+        else {
+            arr.push(['hot',0]);}
+
+        // Create the data table.
+        var data = new google.visualization.DataTable();
+        data.addColumn('string', 'Topping');
+        data.addColumn('number', 'Slices');
+        data.addRows(arr);
+
+        // Set chart options
+        var options = {'title':'Pizza-Piechart',
+                       'width':500,
+                       'height':500};
+
+        var chart = new google.visualization.PieChart(document.getElementById('chart_div'));
+        chart.draw(data, options);
+        var options = {
+          title: 'Pizza-Donut chart',
+          pieHole: 0.5,
+        };
+
+        var chart = new google.visualization.PieChart(document.getElementById('donutchart'));
+        chart.draw(data, options);
+      }
+    </script>
+<style> 
+
+.dinline{display:inline;}
+</style>
+</head>
+<body>
+  <form action="pizza.html" method="get">
```

Comment:
We'll handle the results of the from with jquery, not through a traditional submit form request. I would probably just completely remove the form.